nixos/modules/system/vars/options.nix

```diff
@@ -0,0 +1,185 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+{
+  options.vars = {
+    settings = {
```

Comment:
Not RFC 42. What's the thought behind this name?

Reply:
I understand it's the provider-specific settings, similar in spirit to [how SelfHostBlocks does it](https://github.com/ibizaman/selfhostblocks/blob/c2148eda7704bdb6aa097df05ec5aac0d2285b2b/modules/contracts/default.nix#L16-L23), except here it's (IMO correctly) in terms of modules.